etc/logsets.json

```diff
@@ -1,4 +1,20 @@
 [
+  {
+    "name": "hagfish_logs",
+    "search_dirs": [ "/var/log/usage" ],
+    "regex": "^/var/log/usage/([0-9]+)-([0-9]+)-([0-9]+)T([0-9]+)\\.log\\.gz$",
+    "manta_path": "/%u/stor/sdc/usage/%d/%n/#y/#m/#d/#H.log.gz",
```

Comment:
This matches the one you had @travispaul fix too, right?

Reply:
Looks good to me and matches what's currently in manta, `%d` is the datacenter and `%n` is the CN UUID.